source/guides/1.0/getting-started.md

```diff
@@ -474,7 +474,7 @@ We can use migrations to apply changes to our database schema.
 
 ### Displaying Dynamic Data
 
-With our new experience modelling data, we can get to work displaying dynamic data on our book listing page.
+With our new experience modeling data, we can get to work displaying dynamic data on our book listing page.
```

Comment:
here i realised the guide is using american english, so using one type of english everywhere only makes sense.